app/models/project.rb

```diff
@@ -48,8 +48,7 @@ class Project < ActiveRecord::Base
 
   # Status values can be found at Loan.status_option_set.options and
   # BasicProject.status_option_set.options
-  ACTIVE_STATUSES = %w(active changed possible prospective refinanced relationship
-    relationship_active)
+  STATUSES = %w(active changed possible prospective)
```

Comment:
Maybe `OPEN_STATUSES`. There are other statuses not included here like "complete", "refinanced", and "relationship". I'd choose a name that makes it clear that this is a subset of all the existing status options.